sklearn/preprocessing/data.py

```diff
@@ -525,8 +531,8 @@ class StandardScaler(BaseEstimator, TransformerMixin):
     Attributes
     ----------
     scale_ : ndarray or None, shape (n_features,)
-        Per feature relative scaling of the data. Equal to ``None`` when
-        ``with_std=False``.
+        Per feature relative scaling of the data. Computed using
```

Comment:
"Computed using" is ambiguous. Better to just say "Equal to".

Reply:
Hi @robert-dodier and @eamanu, thank you for your suggestions. I ended choosing the `This is calculated using` to not repeat the `Equal to` in the next sentence. Let me know what you guys think.